### ISSUE TITLE
Types are not exported in index

### ISSUE THREAD

**Issue by Mrtenz** [gilbarbara/react-inlinesvg#151]
The types defined in `types.ts` aren't exported in `index.tsx`. It would be nice if the types were exported there, so you can import it like
```ts
import { Props } from 'react-inlinesvg';
```
instead of
```ts
import { Props } from 'react-inlinesvg/lib/types';
```

**Comment by gilbarbara** [comment]
Feel free to submit a PR

**Comment by gilbarbara** [comment]
Hey

I have exported the types in the index.d.ts declaration.
It's available in 2.0.1
